src/Utils/Parser.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Utils;
+
+use Parsedown;
+
+class Parser {
```

Comment:
Should probably add some comments but seems pretty self explanatory and I don't expect it to change much.